docs/unittesting.md

```diff
@@ -16,23 +21,19 @@ Run Tests
 
 This executes all tests whose box has been checked below (by default all). The execution is run in a separate environment and the result is displayed below.
 
-Here is a list of functions and their supported types that you can use to write your testcases:
-
-| Available functions  | Supported types |
-
-| ------------- | ------------- |
+![Unit Testing Section](images/a-unit-testing-run-result.png)
 
-| `Assert.ok()`  | `bool`  |
-
-| `Assert.equal()`  | `uint`, `int`, `bool`, `address`, `bytes32`, `string`  |
-
-| `Assert.notEqual()` | `uint`, `int`, `bool`, `address`, `bytes32`, `string`  |
-
-| `Assert.greaterThan()` | `uint`, `int` |
+Here is a list of functions and their supported types that you can use to write your testcases:
 
-| `Assert.lesserThan()` | `uint`, `int` |
+| Available functions    | Supported types |
+| ---------------------- | --------------- |
+| `Assert.ok()`          | `bool`          |
+| `Assert.equal()`       | `uint`, `int`, `bool`, `address`, `bytes32`, `string`  |
+| `Assert.notEqual()`    | `uint`, `int`, `bool`, `address`, `bytes32`, `string`  |
+| `Assert.greaterThan()` | `uint`, `int`   |
+| `Assert.lesserThan()`  | `uint`, `int`   |
 
-see [https://github.com/ethereum/remix/blob/master/remix-tests/tests/examples_4/SafeMath_test.sol](https://github.com/ethereum/remix/blob/master/remix-tests/tests/examples_4/SafeMath_test.sol) for some code sample
+Click [here](https://github.com/ethereum/remix/blob/master/remix-tests/tests/examples_4/SafeMath_test.sol) for some code sample.
```

Comment:
Should be:
Click [here](https://github.com/ethereum/remix/blob/master/remix-tests/tests/examples_4/SafeMath_test.sol) for a test file example.

Reply:
Done.